tgui/packages/tgui/interfaces/CommandReport.tsx

```diff
@@ -22,14 +22,15 @@ type Data = {
   custom_name: string;
   played_sound: string;
   print_report: string;
+  append_update_name: boolean;
```

Comment:
use `BooleanLike` here instead (not using a suggestion bc you'll have to import it)